### PR TITLE
feat: extend kanpai pandas metadata

### PR DESCRIPTION
**Pull request by jasonyjong**

### CODE REVIEW

File: packages/indexer/src/metadata/extend/kanpai-pandas/index.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { TokenMetadata } from "@/metadata/types";
+
+const metadataBaseURI = "https://metadata-api-snag-solutions-ad3b2d55c570.herokuapp.com/api/token";
```
Comment (nofir) [inline packages/indexer/src/metadata/extend/kanpai-pandas/index.ts:4]: @jasonyjong The url does not seem to work from my tests. (times out)

Comment (jasonyjong) [reply packages/indexer/src/metadata/extend/kanpai-pandas/index.ts:4]: Hmm works for me now -- I wonder if it's cause heroku spins down their instance. Will message them Kanpai team.

**Review by nofir — APPROVED** [review]

File: packages/indexer/src/metadata/extend/kanpai-pandas/index.ts (hunk 1 above)
Comment (nofir) [reply packages/indexer/src/metadata/extend/kanpai-pandas/index.ts:4]: let me know if I should wait with merging it incase you need to make a change.

Comment (jasonyjong) [reply packages/indexer/src/metadata/extend/kanpai-pandas/index.ts:4]: We should be good to merge -- will just have their team look into it. It should be up for at least 24 hours, but that's only if 0 requests are sent in that time period

**Pull request merged**